README.md

```diff
@@ -108,6 +107,28 @@ ansible-playbook -i ./BraveButtonsConfig/ansible/<inventory file name> \
                  --ask-vault-pass
 ```
 
+# How to release a new version & update the entire Button system (server and RPi fleet)
+
+1. decide on an appropriate version number for the new version
+
+1. update CHANGELOG.md by moving everything in `Unreleased` to a section for the new version
+
+1. make a new git tag in both the `BraveButtons` and `BraveButtonsConfig` repositories with the version number
+
+1. update the `production` branch, ensuring that it is in sync with the tag for the new version
+
+1. ssh into the Button server
+
+1. cd into the `BraveButtons/server` directory
+
+1. shut down the server process: `sudo pm2 stop BraveServer`
+
+1. update the RPi fleet as described above (in a separate ssh session)
+
+1. pull the latest production code: `git checkout production && git pull origin production`
+
+1. run the server setup script: `sudo ./setup_server.sh ./.env`
+
```

Comment:
Is it worth adding a verification step at the end to check on both dashboards to see that everything looks as expected?